src/types.d.ts

```diff
@@ -64,12 +70,12 @@ export interface PaginatedRequestOptions extends BaseRequestOptions {
 }
 
 // Access Requests
-export enum AccessLevel = {
-  GUEST: 10,
-  REPORTER: 20,
-  DEVELOPER: 30,
-  MASTER: 40,
-  OWNER: 50,
+export enum AccessLevel {
+  GUEST= 10,
+  REPORTER= 20,
+  DEVELOPER = 30,
+  MASTER = 40,
```

Comment:
in GitLab v11 they renamed it to Maintainer instead of Master.
https://gitlab.com/gitlab-org/gitlab-ce/issues/42751

Reply:
Nice catch! Ill update it